src/pages/task/taskTabs/testsTable/TestsTableCore.tsx

```diff
@@ -198,87 +260,80 @@ const columnsTemplate: ColumnProps<TestResult>[] = [
   },
   {
     title: <span data-cy="logs-column">Logs</span>,
-    width: 150,
+    width: 230,
     dataIndex: "logs",
     key: "logs",
     sorter: false,
-    render: ({
-      htmlDisplayURL,
-      rawDisplayURL,
-    }: {
-      htmlDisplayURL: string;
-      rawDisplayURL: string;
-    }): JSX.Element => (
-      <>
-        {htmlDisplayURL && (
-          <ButtonWrapper>
+    render: (a, b): JSX.Element => {
+      const { execution, groupID, lineNum, taskId, testFile, logTestName } =
+        b || {};
+      const { htmlDisplayURL, rawDisplayURL } = b?.logs ?? {};
+      const lobsterLink = getLobsterTestLogUrl(
+        taskId,
+        execution,
+        logTestName || testFile,
+        groupID,
+        lineNum
+      );
+
+      return (
+        <>
+          {htmlDisplayURL && !isLobsterLink(htmlDisplayURL) && lobsterLink && (
+            <ButtonWrapper>
+              <Button
+                data-cy="test-table-lobster-btn"
+                size="small"
+                target="_blank"
+                variant="default"
+                href={lobsterLink}
+                onClick={() =>
+                  taskAnalytics.sendEvent({
+                    name: "Click Logs Lobster Button",
+                  })
+                }
+              >
+                Lobster
+              </Button>
+            </ButtonWrapper>
+          )}
+          {htmlDisplayURL && (
+            <ButtonWrapper>
```

Comment:
I think this logic is a little flawed. 
If `isLobsterLink(htmlDisplayURL)` SHOW lobster button
if `isLobsterLink(htmlDisplayURL)` DO NOT SHOW html button

These both link to lobster currently, its confusing to have 2 buttons that both go to the same location but have different names.

Reply:
ah, I realized that `isLobsterLink` was broken and I just updated it.

Reply:
I updated the isLobsterLink function so the conditional render should make more sense. 
check out [this page](http://localhost:3000/task/mongodb_mongo_master_enterprise_rhel_80_64_bit_dynamic_required_aggregation_disabled_optimization_patch_e9f91c18323784288cd89cbbb70bc0eca8e89aae_602ff95457e85a5721939bc6_21_02_19_17_46_09/tests?execution=0&sortBy=STATUS&sortDir=ASC
) which only has two log buttons (Lobster, Raw) as opposed to 3 (Lobster, HTML, Raw). We show only 2 buttons when the HTMLDisplayURL field for the testlog point to lobster instead of the evergreen html viewer.